### DISCUSSION TITLE
How to use  sequelize orm  Deno RestAPI application any tutorials or document?  why this libray is not compatible in deno apps

### DISCUSSION THREAD

**Discussion by rsurendraidexcel** [sequelize/sequelize#16315] [Help & Questions]
How to use  sequelize orm  Deno RestAPI application any tutorials or documents?  why this libray is not compatible in deno apps

any body know please help ?

**Comment by WikiRik** [comment]
Support for Deno is not on the roadmap of the current maintainers, but we are willing to review PRs from the community that improve Deno support.

See https://github.com/sequelize/sequelize/issues/12836
